### DISCUSSION TITLE
Please update IPAdapter

### DISCUSSION THREAD

**Discussion by Gushousekai195** [lllyasviel/stable-diffusion-webui-forge#869] [Ideas]
Since Forge appears to be revived now, I felt like saying...

The regular Controlnet extension for the original webui has advanced weight scheduling which you can edit, and which allows for stuff like style transfer. The built-in one for Forge still doesn't have it.

Can you please update accordingly? I appreciate it.

**Comment by hakuma26** [comment]
+1

**Comment by ali0une** [comment]
why don't you just add this to the list?
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/932

**Comment by gohan2091** [comment]
+1
